control/controldisplay/range.go

```diff
@@ -0,0 +1,33 @@
+package controldisplay
+
+type Range struct {
+	minimum int
+	maximum int
+}
+
+// Constrain a number to be within a range.
+// Returns:
```

Comment:
I think we can do without the explanation - `Constrain a number to be within a range.` is descriptive enough